drivers/hmis/app/graphql/types/application/user_dashboard.rb

```diff
@@ -0,0 +1,70 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class Application::UserDashboard < Types::BaseObject
+    # Type for resolving data needed on the HMIS user dashboard.
+    # Underlying object is Hmis::User, who should always be the currently logged-in user.
+    description 'Resolves everything that is needed on the user dashboard'
+
+    field :id, ID, null: false
+    field :user_dashboard_config, Types::Application::UserDashboardConfig, null: false
+    field :staff_assignments, HmisSchema::StaffAssignment.page_type, null: true
+    field :ce_referral_steps, HmisSchema::CeReferralStep.page_type, null: true
+
+    def user_dashboard_config
+      {
+        id: object.id,
+        show_staff_assignment: show_staff_assignment,
+        show_referrals: show_referrals,
+      }
+    end
+
+    def staff_assignments
+      object.staff_assignments.
+        viewable_by(current_user).
+        open_on_date. # This will include households that exited today
+        order(created_at: :desc, id: :desc)
```

Comment:
Scope-creeping here, but it is inconsistent that we sort households this way. It can result in "incomplete" households appearing below "active" households, which we don't do by default anywhere else.


Suggest adjusting this to use the existing household recency scope instead of sorting by assignment date:
```suggestion
        merge(Hmis::Hud::Household.sort_by_option(:most_recent))
```